native/cocos/audio/android/AudioPlayerProvider.cpp

```diff
@@ -40,11 +39,21 @@ THE SOFTWARE.
 #include "audio/android/utils/Utils.h"
 #include "base/memory/Memory.h"
 #include "base/ThreadPool.h"
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
+#include <sys/system_properties.h>
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+#include "cocos/platform/FileUtils.h"
+#include "cocos/platform/openharmony/FileUtils-OpenHarmony.h"
```

Comment:
Same as OpenSLES, should be set in a single file, don't make it hard to read

Reply:
I don't think it is necessary